### PR TITLE
[Test] Validate OSD blocklist clients

### PR DESCRIPTION
**Pull request by sayaleeraut**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by sayaleeraut** [comment]
Tox execution results -

==================================================== 17 passed, 5 warnings in 0.35s ====================================================
py3: commands[2]> flake8
py3: commands[3]> isort -c .
Skipped 3 files
py3: commands[4]> black --check --diff .
All done! ✨ 🍰 ✨
1131 files would be left unchanged.
py3: commands[5]> yamllint -d relaxed --no-warnings .
.pkg: _exit> python /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
  py3: OK (32.40=setup[3.89]+cmd[2.48,0.50,1.55,1.43,3.23,19.32] seconds)
  congratulations :) (32.42 seconds)

**Review by MohitBis — APPROVED** [review]
LGTM

File: suites/reef/cephadm/tier2-osd-scenarios.yaml [hunk 1]
```diff
@@ -152,3 +152,10 @@ tests:
       module: test_verify_incorrect_mem_usage.py
       polarion-id: CEPH-83584007
       abort-on-fail: true
+
+  - test:
```
Comment (MohitBis) [inline suites/reef/cephadm/tier2-osd-scenarios.yaml:156]: Hi @sayaleeraut 

Can we add same test from squid as well.

Rest look good

Thanks,
Mohit

**Comment by vinpapnoi** [comment]
@sayaleeraut please add test execution logs

**Review by pranavprakash20 — DISMISSED** [review]
Lgtm

File: suites/reef/cephadm/tier2-osd-scenarios.yaml (hunk 1 above)
Comment (sayaleeraut) [reply suites/reef/cephadm/tier2-osd-scenarios.yaml:156]: Done @MohitBis , I have added the test in the Quincy and Squid suite as discussed.

**Comment by sayaleeraut** [comment]
Execution logs from Reef - 
https://ocs3xstage-jenkins-csb-rhgsocs3x.apps.ocp-c1.prod.psi.redhat.com/view/Executor/job/qe-reef-test-executor-openstack/147/

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by adityaramteke — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by vinpapnoi — APPROVED** [review]
LGTM

**Review by tintumathew10 — APPROVED** [review]
lgtm

**Pull request merged**